common/beerocks/bwl/dwpal/base_wlan_hal_dwpal.cpp

```diff
@@ -658,12 +658,21 @@ size_t base_wlan_hal_dwpal::dwpal_nl_cmd_get(const std::string &ifname, unsigned
         return nl_handler_cb_wrapper(ifname, event, subevent, len, data);
     };
 
-    //parsing will be done in callback func
-    if (dwpal_driver_nl_msg_get(m_dwpal_nl_ctx, DWPAL_NL_SOLICITED_EVENT, nl_handler_cb, NULL) ==
-        DWPAL_FAILURE) {
-        LOG(ERROR) << " dwpal_driver_nl_msg_get failed,"
-                   << " ctx=" << m_dwpal_nl_ctx;
-        return data_size;
+    // Since we're expecting a Solicited (asynchronous) event from the driver,
```

Comment:
Commit message:
>also, added 1 sec TO is added so we dont get stuck here forever.
In practice, the response usually arrives much faster.

Should change to

>also, added 1-second timeout is added so we don't get stuck here forever.
In practice, the response usually arrives much faster.